### PR TITLE
Fixed CI

### PR DESCRIPTION
**Pull request by gnought**
For https://github.com/moscajs/aedes/issues/371, the Travis CI is broken after migrating to moscajs org. We now switch to use Github Actions.

### CODE REVIEW

**Comment by gnought** [comment]
faucet is a 6-year-old project and its dependency sprintf is deprecated.
pre-commit is a 3-year-old project and it seems it is dead, our CI has covered tests.

**Comment by mcollina** [comment]
please replace pre-commit with husky, this is useful. Also, does faucet cause any issue? There are a few other renders to be used in dev.
Il 26 gen 2020, 14:40 +0100, Gnought <notifications@github.com>, ha scritto:

**Comment by gnought** [comment]
Husky requires Node >= 10, should our project drops Node v8 support?
hmm, we could put back pre-commit and faucet back and try to establish a proper github actions, and let see if we get any errors, agree?

**Comment by mcollina** [comment]
that's good, ok
Il 26 gen 2020, 15:13 +0100, Gnought <notifications@github.com>, ha scritto:

File: package.json [hunk 1]
```diff
@@ -73,14 +69,13 @@
...
     "convert-hrtime": "^3.0.0",
-    "coveralls": "^3.0.9",
     "duplexify": "^4.1.1",
     "faucet": "0.0.1",
     "license-checker": "^25.0.1",
     "mqtt": "^3.0.0",
     "mqtt-connection": "^4.0.0",
     "nyc": "^15.0.0",
-    "pre-commit": "^1.2.2",
+    "precommit": "^1.2.2",
```
Comment (mcollina) [inline package.json]: Can you restore this?

File: package.json [hunk 2]
```diff
@@ -8,19 +8,15 @@
...
-    "coveralls": "cat coverage/lcov.info | coveralls",
-    "license-checker": "license-checker --production --onlyAllow='MIT;ISC;BSD-3-Clause;BSD-2-Clause'"
+    "test:ci": "npm run lint && npm run typescript && npm run coverage",
+    "license-checker": "license-checker --production --onlyAllow=\"MIT;ISC;BSD-3-Clause;BSD-2-Clause\""
   },
   "pre-commit": [
-    "lint",
-    "test",
-    "typescript-test"
+    "test"
```
Comment (mcollina) [inline package.json]: fix these

Comment (gnought) [reply package.json]: The npm run test has been changed to
```
    "test": "npm run lint && npm run unit && npm run typescript",
```

**Pull request merged**